wye/workshops/mixins.py

```diff
@@ -1,5 +1,19 @@
 from wye.profiles.models import Profile
 from wye.base.emailer import send_mail
+from .models import Workshop
+from django.core.exceptions import PermissionDenied
+
+
+class WorkshopAccessMixin(object):
+
+    def dispatch(self, request, *args, **kwargs):
+        pk = self.kwargs.get(self.pk_url_kwarg, None)
+        workshop = Workshop.objects.get(id=pk)
+        if workshop.requester not in self.request.user.organisation_users.all():
+            print(workshop.requester)
+            print(self.request.user.organisation_users.all())
```

Comment:
I had planned to remove it with my next commit, had forgotten to remove the debug messages last time.